src/base.rs

```diff
@@ -84,8 +94,31 @@ extern "C" {
     pub static kCFAllocatorNull: CFAllocatorRef;
     pub static kCFAllocatorUseContext: CFAllocatorRef;
 
+    pub fn CFAllocatorGetTypeID() -> CFTypeID;
+
+    pub fn CFAllocatorSetDefault(allocator: CFAllocatorRef);
+    pub fn CFAllocatorGetDefault() -> CFAllocatorRef;
+
+    pub fn CFAllocatorCreate(allocator: CFAllocatorRef, context: *mut CFAllocatorContext) -> CFAllocatorContext;
+    pub fn CFAllocatorAllocate(allocator: CFAllocatorRef, size: CFIndex, hint: CFOptionFlags) -> *mut c_void;
+    pub fn CFAllocatorReallocate(allocator: CFAllocatorRef, ptr: *mut c_void, newSize: CFIndex, hint: CFOptionFlags) -> *mut c_void;
+    pub fn CFAllocatorDeallocate(allocator: CFAllocatorRef, ptr: *mut c_void);
+    pub fn CFAllocatorGetPreferredSizeForSize(allocator: CFAllocatorRef, size: CFIndex, hint: CFOptionFlags) -> CFIndex;
+    pub fn CFAllocatorGetContext(allocator: CFAllocatorRef, context: *mut CFAllocatorContext);
+
     pub fn CFGetTypeID(cf: CFTypeRef) -> CFTypeID;
+    pub fn CFCopyTypeIDDescription(cf: CFTypeID) -> CFStringRef;
 
     pub fn CFRetain(cf: CFTypeRef) -> CFTypeRef;
     pub fn CFRelease(cf: CFTypeRef);
+    pub fn CFAutorelease(arg: CFTypeRef) -> CFTypeID;
```

Comment:
I don't think it is - it looks like it's an attribute to say that an argument is sent a `release` message when the function is called (https://clang-analyzer.llvm.org/annotations.html).